documentation/src/docs/security/securing-distributed-mode-cluster.md

```diff
@@ -0,0 +1,311 @@
+
+| Files | Example File| Description |
+|:-----:| :---:|:--------|
+| Certificate in PEM format | `controller01.pem` file | The signed certificate to be used by the service.|
+| Private key in PEM format | `controller01.key.pem` file  |  The private key to be used by the service. |
+| Server keystore in JKS format | `controller01.server.jks` file | The keystore file to be used by the service. |
+
+### Deploying certificates and enabling TLS in Pravega
+
+We'll discuss this in the section titled Enabling Security in Pravega, together with other security configuration and setup.
+
+## Enabling Security in Pravega
+
+Here, we discuss about these tasks:
+
+1. Configuring TLS and Auth Parameters on the Server
+2. Configuring TLS and credentials on the client
+3. Having the TLS and Auth parameters take effect
+
+
+### Configuring TLS and Auth Parameters on the Server
+
+This task is about using the certificates, keys, keystores and truststores to configure TLS on the server side, and configuring Auth.
+
+We'll need to configure the following services on the server side:
+
+1. Controller services
+2. Segment Store services
+3. Zookeeper services (Optionally)
+
+Lets' discuss about #1 and #2 below. For information about turning TLS on for Zookeeper, refer to its [SSL](https://cwiki.apache.org/confluence/display/ZOOKEEPER/ZooKeeper+SSL+User+Guide) documentation.
+
+**Controller Service:**
+
+Controller services can be configured in three distinct ways:
+
+1. By specifying the configuration parameter values directly in the `controller.config.properties` file.
+
+   ```
+   controller.auth.tlsEnabled=true
+   controller.auth.tlsCertFile=/etc/secrets/controller01.pem
+   ```
+2. By specifying configuration parameters via corresponding environment variables. For example,
+
+   ```
+   # TLS_ENABLED environment variable corresponds to Controller configuration parameter
+   # "controller.auth.tlsEnabled".
+   $ export TLS_ENABLED: "true"
+   ```
+
+   To identify the environment variables corresponding to the configuration parameter, inspect the default `controller.config.properties` file and locate the variables.
+
+   ```
+   controller.auth.tlsEnabled=${TLS_ENABLED}
+   controller.auth.tlsCertFile=${TLS_CERT_FILE}
+   ```
+
+3. By specifying configuration parameters (such as `controller.auth.tlsEnabled`) as JVM system properties. This is not applicable for manual deployment and is relevant for container application deployment tools and orchestrators such as Docker Compose, Swarm and Kubernetes.
+
+
+The table below lists the Controller services' TLS and auth parameters and representative values for quick reference. For a detailed discription of these parameters refer to [this](https://github.com/pravega/pravega/blob/master/documentation/src/docs/security/pravega-security-configurations.md) document.
```

Comment:
remove `

Reply:
Removed the code syntax formatting for the configuration property, as suggested. 

Not sure what you mean by "check the auth format". If you are referring to auth as in authentication and authorization, I'm now using the same format everywhere, i.e. without putting it inside a code block.